src/App.tsx

```diff
@@ -1,19 +1,17 @@
 import React, { Component } from "react";
 import { BrowserRouter, Route, Switch } from "react-router-dom";
 
+// tslint:disable: ordered-imports
```

Comment:
What is the reason you disabled the ordered-imports?

Reply:
I intend the orders of the imports to follow the order of the navigation/react-router listing so you can find the specific component easier.

Reply:
Okay, this is a reason I can agree on